x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/prebuilt_rules/prebuilt_rule_customization/trial_license_complete_tier/rules_export.ts

```diff
@@ -46,14 +46,14 @@ export default ({ getService }: FtrProviderContext): void => {
     });
 
     it('exports a set of custom installed rules via the _export API', async () => {
-      await securitySolutionApi
-        .bulkCreateRules({
-          body: [
-            getCustomQueryRuleParams({ rule_id: 'rule-id-1' }),
-            getCustomQueryRuleParams({ rule_id: 'rule-id-2' }),
-          ],
-        })
-        .expect(200);
+      await Promise.all([
+        securitySolutionApi
+          .createRule({ body: getCustomQueryRuleParams({ rule_id: 'rule-id-1' }) })
+          .expect(200),
+        securitySolutionApi
+          .createRule({ body: getCustomQueryRuleParams({ rule_id: 'rule-id-2' }) })
+          .expect(200),
+      ]);
```

Comment:
I'm thinking if these changes to rules export tests should go in a separate PR so that we could backport them to `8.x` to decrease the chance of introducing conflicts with any other follow-up PRs related to rule import/export that will be merged to `8.x`.

Reply:
Ok, I'll open a new PR against `8.x` after merging this one 👍

Reply:
@dplumlee Here we go, looking for a quick 👍 : https://github.com/elastic/kibana/pull/198953